app/models/conference_preference.rb

```diff
@@ -1,5 +1,21 @@
 class ConferencePreference < ActiveRecord::Base
+  validates :terms_accepted?, acceptance: true
```

Comment:
Right, but I don't see a problem with this. You can still add things like "only on create", etc. to the validation. It's still a normal Rails-validation.